R/FilterStates.R

```diff
@@ -385,7 +385,8 @@ FilterStates <- R6::R6Class( # nolint
                 observeEvent(
                   eventExpr = fs_callback(), # when remove button is clicked in the FilterState ui
                   once = TRUE, # remove button can be called once, should be destroyed afterwards
-                  handlerExpr = private$state_list_remove(state$get_state()$id)
+                  handlerExpr = private$state_list_remove(state$get_state()$id),
```

Comment:
```suggestion
                  handlerExpr = private$state_list_remove(state$get_id()),
```

Reply:
That would be cleaner :+1: 

There's just 1 problem as the `get_id()` call is a private method, so this won't work

Are you suggesting we move it to the public methods?

Reply:
Oh no, my mistake leave it as is :)